apps/backend/src/applications/application.entity.ts

```diff
@@ -18,13 +18,21 @@ import {
 import { GetApplicationResponseDTO } from './dto/get-application.response.dto';
 import { Review } from '../reviews/review.entity';
 import { GetAllApplicationResponseDTO } from './dto/get-all-application.response.dto';
+import { FileUpload } from '../file-upload/entities/file-upload.entity';
+import { PrimaryGeneratedColumn } from 'typeorm';
 
 @Entity()
 export class Application {
-  @Column({ primary: true, generated: true })
+  @PrimaryGeneratedColumn()
```

Comment:
lets add migrations for these things since multiple people are making schema changes at the same time so that theres an automated way to keep everyone in sync.

@Atrya537 is also writing some so could be good to sync

[More Info](https://orkhan.gitbook.io/typeorm/docs/migrations)